doc/content/source/postprocessors/Reactivity.md

```diff
@@ -0,0 +1,23 @@
+# Reactivity
+
+!syntax description /Postprocessors/Reactivity
+
+## Description
+
+This postprocessor extracts the combined $k$ eigenvalue from the latest OpenMC eigenvalue
+calculation and calculates reactivity as:
+\begin{equation}
+\label{line-eq2}
+reactivity = \frac{k_eff-1}{k_eff}
```

Comment:
```suggestion
reactivity = \frac{k-1}{k}
```

Reply:
When you do a multi-character subscript in latex, you need to put all characters inside curly braces or else it will only put the first character in the subscript. I also prefer to just write `k` instead of k-effective.